service/src/main/java/bio/terra/catalog/rawls/RawlsService.java

```diff
@@ -99,4 +103,44 @@ public SystemStatusSystems status() {
     }
     return result;
   }
+
+  EntitiesApi entitiesApi(AuthenticatedUserRequest user) {
+    return new EntitiesApi(getApiClient(user));
+  }
+
+  public void exportWorkspaceDataset(
+      AuthenticatedUserRequest user, String workspaceIdSource, String workspaceIdDest) {
+    try {
+      // build source name
+      WorkspaceDetails workspaceDetailsSource =
+          workspacesApi(user).getWorkspaceById(workspaceIdSource, List.of()).getWorkspace();
+      WorkspaceName workspaceNameSource =
+          new WorkspaceName()
+              .namespace(workspaceDetailsSource.getNamespace())
+              .name(workspaceDetailsSource.getName());
```

Comment:
Would it be worth making a static method that converted from `WorkspaceDetails` to `WorkspaceName` to avoid the code duplication?